specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-06-01/machineLearningServices.json

```diff
@@ -0,0 +1,4073 @@
+          "Workspaces"
+        ],
+        "description": "Lists all the available machine learning workspaces under the specified subscription.",
+        "operationId": "Workspaces_ListBySubscription",
+        "x-ms-examples": {
+          "Get Workspaces by subscription": {
+            "$ref": "./examples/getWorkspacesBySubscription.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/PaginationParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the request was well-formed and received properly.",
+            "schema": {
+              "$ref": "#/definitions/WorkspaceListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/connections": {
+      "get": {
+        "tags": [
+          "WorkspaceConnections"
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "x-ms-examples": {
+          "ListWorkspaceConnections": {
+            "$ref": "./examples/listWorkspaceConnections.json"
+          }
+        },
+        "operationId": "WorkspaceConnections_List",
+        "description": "List all connections under a AML workspace.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/TargetParameter"
+          },
+          {
+            "$ref": "#/parameters/CategoryParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the connections.",
+            "schema": {
+              "description": "The response includes a paginated array of Workspace connections and a URI to the next set of results, if any. For the more information the limits of the number of items in a resource group, see https://azure.microsoft.com/en-us/documentation/articles/azure-subscription-service-limits/.",
+              "schema": {
+                "$ref": "#/definitions/PaginatedWorkspaceConnectionsList"
+              }
+            }
```

Comment:
There are two levels of `schema` in this response, which seems problematic to me, could you please have a look?
This may be the source of the autorest failure

Reply:
Thank you Dapeng very much for your suggestion. That is the source of the LintDiff failure. Now it's resolved!